.eslintrc.js

```diff
@@ -0,0 +1,220 @@
+const NodePlugin = require('./tools/node_modules/eslint-plugin-node');
+NodePlugin.RULES_DIR = 'tools/eslint-rules';
+
+const Module = require('module');
+const path = require('path');
+
+const ModuleFindPath = Module._findPath;
+const hacks = ['eslint-plugin-node', 'babel-eslint'];
+Module._findPath = (request, paths, isMain) => {
+  const r = ModuleFindPath(request, paths, isMain);
+  if (!r && hacks.includes(request))
+    return require.resolve(`./tools/node_modules/${request}`);
+  return r;
+};
```

Comment:
As an alternative to monkeypatching the module resolution code, you could also use something like [`eslint-plugin-rulesdir`](https://github.com/not-an-aardvark/eslint-plugin-rulesdir).

Reply:
i actually started by looking at things like that. eslint-plugin-rulesdir solves a different problem. the eslint-plugin-node plugin actually does the same thing as eslint-plugin-rulesdir. the reason i monkeypatch is because eslint doesn't know (rightfully so) to look in `tools/node_modules` for babel-eslint and eslint-plugin-node